src/repr/src/row.rs

```diff
@@ -482,21 +482,49 @@ enum Tag {
     NegativeInt64_48,
     NegativeInt64_56,
     NegativeInt64_64,
+
+    // These are like the ones above, but for unsigned types. The
+    // situation is slightly simpler as we don't have negatives.
+    UInt8_0, // i.e., 0
+    UInt8_8,
+
+    UInt16_0,
+    UInt16_8,
+    UInt16_16,
+
+    UInt32_0,
+    UInt32_8,
+    UInt32_16,
+    UInt32_24,
+    UInt32_32,
+
+    UInt64_0,
+    UInt64_8,
+    UInt64_16,
+    UInt64_24,
+    UInt64_32,
+    UInt64_40,
+    UInt64_48,
+    UInt64_56,
+    UInt64_64,
 }
 
 impl Tag {
     fn actual_int_length(self) -> Option<usize> {
```

Comment:
In there a reason for why we chose to not add an `#[inline]` annotation here?

Reply:
No, I haven't measured it either way.

Reply:
Definitely not a blocker for merging, so please feel free to look into this item afterwards if it is better time-wise.